### PR TITLE
Added set transaction result

### PR DESCRIPTION
**Pull request by Alexandr1992**
Added set transaction result according to the issue https://github.com/O3Labs/O3Android/issues/303

### CODE REVIEW

**Review by xxxdens — APPROVED** [review]
What about case when transaction was declined?

**Comment by Alexandr1992** [comment]
> What about case when transaction was declined?

Added

File: app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendSuccessFragment.kt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package network.o3.o3wallet.Wallet.SendV2
 
 
+import android.app.Activity
```
Comment (xxxdens) [inline app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendSuccessFragment.kt]: Looks like unused import

File: app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt [hunk 2]
```diff
@@ -86,14 +88,35 @@ class   SendReviewFragment : Fragment() {
...
+        val intent = Intent()
+        intent.putExtra("assetName", sendViewModel.getSelectedAsset().value!!.name)
+        intent.putExtra("amount", sendViewModel.getSelectedSendAmount().toDouble())
+        intent.putExtra("address", sendViewModel.getSelectedAddress().value!!)
+
+        if (sendViewModel.txID != null){
+            intent.putExtra("transactionId", sendViewModel.txID)
+        }
+
+        if (sendResult != null){
```
Comment (xxxdens) [inline app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt]: Here and in 97 line there are not enough space before brace
Just to follow the code style in file

File: app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt [hunk 3]
```diff
@@ -86,14 +88,35 @@ class   SendReviewFragment : Fragment() {
...
     }
 
+    fun createSendResultIntent(sendViewModel: SendViewModel, sendResult: String?): Intent {
+        val intent = Intent()
+        intent.putExtra("assetName", sendViewModel.getSelectedAsset().value!!.name)
+        intent.putExtra("amount", sendViewModel.getSelectedSendAmount().toDouble())
+        intent.putExtra("address", sendViewModel.getSelectedAddress().value!!)
+
+        if (sendViewModel.txID != null){
+            intent.putExtra("transactionId", sendViewModel.txID)
```
Comment (xxxdens) [inline app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt]: Looks like this parameter is not set yet. Its value is updated after createSendResultIntent is called in 116 line
And it is actually equals to result

**Review by xxxdens — CHANGES_REQUESTED** [review]

File: app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendSuccessFragment.kt (hunk 1 above)
Comment (Alexandr1992) [reply app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendSuccessFragment.kt]: No, it's for constants Activity.RESULT_OK and RESULT_CANCELED

File: app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt (hunk 2 above)
Comment (Alexandr1992) [reply app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt]: Thanks, I added spaces

File: app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt (hunk 3 above)
Comment (Alexandr1992) [reply app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendReviewFragment.kt]: No, Initialization of txID and sendResult variables is in SendViewModel class on transactions

There are 3 funcs and ways to do transactions in SendViewModel:
- sendOntAsset          - set up only sendResult
- sendNativeNeoAsset    - set up txID and sendResult with same value
- sendNeoTokenAsset     - set up only sendResult

i.e.

if (sendResult == null)
    100% failed
    
if (txID != null)
    100% success

if (txID == null)
    if (sendResult == valid transaction id)
        success
    else
        failed

File: app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendSuccessFragment.kt (hunk 1 above)
Comment (Alexandr1992) [reply app/src/main/java/network/o3/o3wallet/Wallet/SendV2/SendSuccessFragment.kt]: ок

**Review by xxxdens — APPROVED** [review]

**Pull request merged**